tensorflow/core/kernels/BUILD

```diff
@@ -993,7 +993,8 @@ tf_kernel_library(
     visibility = [":friends"],
     deps = [
         ":bounds_check",
-        "//tensorflow/core:framework_lite",
```

Comment:
we probably should not remove that.
Can we wrap our dependency in if_sycl() ? Like done here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/BUILD#L498 
Or is that needed on CPU side as well?